transformers/gtfs.py

```diff
@@ -40,7 +40,8 @@
     UicOperatingPeriodRef,
     OperatingPeriodRef,
 )
-from domain.netex.model.name_of_class_operating_period_ref_structure import NameOfClassOperatingPeriodRefStructure
+#TODO remove as an correct
+from domain.netex.model.name_of_class_operating_period_ref_structure_type import NameOfClassOperatingPeriodRefStructureType as NameOfClassOperatingPeriodRefStructure
```

Comment:
Please elaborate why the renaming is done.

Reply:
@skinkie  it seems that now all are _type in the end. In this file they were not. So i changed the import, but not yet the content of gtfs.py. At least in my version. Or is something wrong with my model again (regenerated it before I started)